preview/Win7Msix/Win7MSIXInstaller/InstallUI.hpp

```diff
@@ -22,6 +22,7 @@ static HWND g_CancelbuttonHWnd = NULL;
 static HWND g_LaunchbuttonHWnd = NULL;
 static bool g_launchCheckBoxState = true; /// launch checkbox is checked by default
 static bool g_installed = false;
+static PCWSTR g_installOrUpdateText = L"Install "; /// Default button and install screen UI text is 'Install'
```

Comment:
>Install [](start = 40, length = 7)

we should be moving these UX elements over to string resources so that they can eventually be localized

Reply:
Moved strings to resource file.